include/lsqecc/ls_instructions/ls_instructions.hpp

```diff
@@ -65,7 +65,7 @@ struct BellPairInit {
     PlaceNexTo loc2;
 
     std::optional<std::vector<LocalInstruction::LocalLSInstruction>> local_instructions;
-    std::optional<unsigned int> counter;
+    std::optional<std::pair<unsigned int, unsigned int>> counter;
```

Comment:
When a LocalInstruction cannot be placed, a new slice is ordered. The pair allows us to track both where we started and left off in the local_instructions vector each time the BellPrepLLI is attempted so that the InstructionVisitor knows what to print for each slice.